### PR TITLE
Setup Topic and two integration guides

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Tom has created integration guides which we want to publish

### Resolves:
 [OKTA-164180](https://oktainc.atlassian.net/browse/OKTA-164180)

### CODE REVIEW

File: _source/_docs/integration-guides/setting-up-okta-for-integration-examples.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+    > Important: In a production environment, use the authorization code flow wherever possible instead of the implicit flow. Also, use real values for the `nonce` and `state` values. For more information, see [OpenID Connect and OAuth 2.0 API](/docs/api/resources/oidc#request-parameters).
+
+5. Test your client and authorization server. Use the URL in the following examples to test your OIDC client and authorization server. 
+
+
+### Example with Actual Values
+
+```
+https://partnerpoc.oktapreview.com/oauth2/ausce8ii5wBzd0zvQ0h7/v1/authorize?
```
Comment (jmelberg-okta) [inline _source/_docs/integration-guides/setting-up-okta-for-integration-examples.md]: I'd consider removing this example in favor of the "**Example with Variables**" section.

We have  moved away from documenting `*.oktapreview` orgs throughout our documentation about a ways back.

Comment (mystiberry-okta) [reply _source/_docs/integration-guides/setting-up-okta-for-integration-examples.md]: But they can actually issue this request to that actual org, I believe, which is the point of the 2nd section. Will double-check with Tom Smith.

Comment (mystiberry-okta) [reply _source/_docs/integration-guides/setting-up-okta-for-integration-examples.md]: for now I'll replace that and add a footnote. We only look for oktapreview in code blocks, @jmelberg-okta ?

Comment (jmelberg-okta) [reply _source/_docs/integration-guides/setting-up-okta-for-integration-examples.md]: Without user credentials, the `/authorize` request doesn't provide any benefit, as the user will land on partnerpoc's main login page. Can you or Tom Smith elaborate a bit more why having this request is useful to the reader?

We `grep` through all of our pages (text, code blocks, HTML, etc.) looking for `*.oktapreview.com`, so we'd have to update our tests to reflect this one-off change.

Comment (mystiberry-okta) [reply _source/_docs/integration-guides/setting-up-okta-for-integration-examples.md]: Ack. If this is an actual org for anyone to use, I'll ping you. Assume it's not for now. Though long-term, never being able to refer to one of our most basic tenants is possibly not the right thing. Not suer.

**Comment by mystiberry-okta** [comment]
@jakubvul I'll fix any errors that tests reveal, but am done developing this content and handing off to you.

**Pull request closed without merging**